app/javascript/packs/form-validation.ts

```diff
@@ -50,25 +52,23 @@ function checkInputValidity(event) {
 /**
  * Binds validation to a given input.
  *
- * @param {HTMLInputElement} input Input element.
+ * @param input Input element.
  */
-function validateInput(input) {
+function validateInput(input: HTMLInputElement) {
   input.addEventListener('input', checkInputValidity);
   input.addEventListener('invalid', checkInputValidity);
 }
 
 /**
  * Initializes validation on a form element.
  *
- * @param {HTMLFormElement} form Form to initialize.
+ * @param form Form to initialize.
  */
-export function initialize(form) {
-  /** @type {HTMLInputElement[]} */
-  const fields = Array.from(form.querySelectorAll('.field,[required]'));
+export function initialize(form: HTMLFormElement) {
+  const fields: HTMLInputElement[] = Array.from(form.querySelectorAll('.field,[required]'));
```

Comment:
another idea about a space
```suggestion
  const fields: HTMLInputElement[] = Array.from(form.querySelectorAll('.field, [required]'));
```

Reply:
Updated in f15bb4059.